server/server.js

```diff
@@ -16,22 +16,22 @@ app.get("/", function(req, res, next) {
     res.sendFile(filePath);
 });
 
-app.use(function(req, res, next) {
```

Comment:
i originally tried putting this function here (i.e. ```app.use(authMiddleware)```, but kept getting an error. maybe because it's an async function? anyway, it works the way i did it...

Reply:
interesting; yeah I don't fully understand the ordering / why 'use' wouldn't work here but as long as it works sounds good